### PR TITLE
Kernel docs updates

### PR DESCRIPTION
**Pull request by mbolivar**
I had occasion to reread through the kernel reference docs while looking at RTIO and have a few trivial patches I think might make life easier for the reader.

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]
Looks nice!

**Review by dbkinder — APPROVED** [review]
We'll wait for the CI check to be sure, but LGTM!  Thanks.

**Review by nashif — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]

**Pull request merged**
